data/json/itemgroups/Locations_MapExtras/locations.json

```diff
@@ -802,7 +802,7 @@
     "id": "recycle_iron",
     "type": "item_group",
     "items": [
-      { "item": "bolt_metal", "prob": 15, "charges-min": 1, "charges-max": 3 },
+      { "item": "nuts_bolts", "prob": 15, "charges-min": 7, "charges-max": 200 },
```

Comment:
I don't know much really about this json, so might be out of my league here, but this `item_group` already seems to contain `nuts_bolts` at line 812. Maybe we should just remove this line.

Reply:
Oh, i miss that one, my bad
```suggestion
```